vitamins/scs_bearing_block.scad

```diff
@@ -0,0 +1,147 @@
+//
+// NopSCADlib Copyright Chris Palmer 2018
+// nop.head@gmail.com
+// hydraraptor.blogspot.com
+//
+// This file is part of NopSCADlib.
+//
+// NopSCADlib is free software: you can redistribute it and/or modify it under the terms of the
+// GNU General Public License as published by the Free Software Foundation, either version 3 of
+// the License, or (at your option) any later version.
+//
+// NopSCADlib is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+// without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with NopSCADlib.
+// If not, see <https://www.gnu.org/licenses/>.
+//
+
+//
+// SCSnUU and SCSnLUU bearing blocks
+//
+
+
+include <NopSCADlib/core.scad>
+include <NopSCADlib/vitamins/screws.scad>
```

Comment:
Vitamins only need to be included if an object name is needed, which is not the case here because screws and bearings come from the scs_bearing object.   It is better to use screw.scad, nut.scad and linear_bearing.scad.

Reply:
OK.